src/recluster.c

```diff
@@ -0,0 +1,1175 @@
+/*-------------------------------------------------------------------------
+ *
```

Comment:
should we do a cluster_import.c/h akin to planner_import where we put pg functions that are completely unmodified? This lets us know right away which parts have not been touched. Which of these functions would qualify if we did that. To be clear: If there are functions we did modify to get rid of code inside we should keep those modified in this file.

Reply:
Just to be clear I don't know if that's a good idea. Asking

Reply:
First I'll have to check and see how much unmodified code is left; the garbage collection we did touched most of it.

Reply:
It looks like none of the functions are untouched except `reform_and_rewerite_tuple`.

Reply:
Ok then nevermind. Let's just keep it all in here.